test/functional/postalcode/postalcode.js

```diff
@@ -0,0 +1,99 @@
+define([
+	"globalize",
+	"json!cldr-data/supplemental/likelySubtags.json",
+	"json!cldr-data/supplemental/postalCodeData.json",
+	"../../util",
+	"json!./dummy/kr.json",
+	"json!./dummy/us.json",
+
+	"globalize/postalcode"
+
+], function( Globalize, likelySubtags, postalcode, util, kr, us ) {
+
+	QUnit.module( ".validatePostalCode( value )", {
+		setup: function() {
+			Globalize.load( likelySubtags, {
+				main: {
+					ko: {}
+				}
+			});
+			Globalize.locale( "ko" );
```

Comment:
For clarity, I'm OK with keeping the alias method `Globalize.validatePostalCode()`. I'm against having the instance methods.